build.cmd

```diff
@@ -1,3 +1,3 @@
 @echo off
-powershell -NoLogo -NoProfile -ExecutionPolicy ByPass -Command "& """%~dp0eng\common\build.ps1""" -build -restore %*"
+powershell -NoLogo -ExecutionPolicy ByPass -NoProfile -command "& """%~dp0eng\common\Build.ps1""" -restore -build -msbuildEngine dotnet %*"
```

Comment:
I don't know whether this will matter or not in actual usage. The build script name is lowercase on disk. Does this cause issues on case-sensitive platforms?

Reply:
If you decide to change this back, please also change restore.cmd and test.cmd

Reply:
Oh

Reply:
I think it will make diff on linux, windows is fine